libr/core/linux_heap_glibc.c

```diff
@@ -304,6 +304,14 @@ static bool GH(r_resolve_symbol)(RCore *core, GHT *symbol, const char *symname)
 		eprintf ("Warning: Can't find glibc mapped in memory (see dm)\n");
 		return false;
 	}
+
+	const char *debug_file = strdup (r_config_get (core->config, "dbg.glibc.dbglib"));
```

Comment:
if you strdup, result is char*, not const. and this will leak.

i dont think you need to strdup at all, just r_config_get